packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -32,16 +36,18 @@ export default function createStudioStylingPlugin(): PluginOption {
   };
 }
 
-function isImportedByStudio(id: string) {
-  return !!(id.match(/.*@yext\/studio.*/) || id.match(/.*packages\/studio*/));
+function isImportedByStudio(importer: string) {
+  return !!(
+    importer.match(/.*@yext[\\\\/]studio.*/) || importer.startsWith(packagesDir)
+  );
 }
 
 function isStyleFile(id: string) {
-  return !!id.match(/.*\.s?css.*/);
+  return !!id.match(/.*\.css.*/);
```

Comment:
is this the same as `id.endsWith(".css")`?

Reply:
I wanted to make sure that it would work on any css file, even if we or Vite happened to add any query parameters (`e.g. ?inline` or `?import`)!

Reply:
got it. maybe the regex can check that there's optionally a `?` followed by any characters at the end? we wouldn't want it to match something like `.../index.css.tsx`. or alternatively, we could split on `?` and check that the first section ends with `.css`

Reply:
went with the latter!